src/lib/utils/match.ts

```diff
@@ -1,6 +1,12 @@
 import { isMatch } from 'micromatch';
 
+function fixMatchString(str: string): string {
+  return str.replace(/\//g, '-');
+}
+
 export default function match(str: string, include: string[], exclude: string[]): boolean {
-  const matchStr = str.replace(/\//g, '-');
-  return (include.length === 0 || isMatch(matchStr, include)) && (exclude.length === 0 || !isMatch(matchStr, exclude));
+  const matchStr = fixMatchString(str)
+  const included = (include || []).map((str) => fixMatchString(str))
```

Comment:
Isn't `include`/`exclude` expected to always be an array? The `|| []` in both of these seem to be redundant.
Also I think `.map(fixMatchString)` would look a little clearer.

Reply:
Good point, I forgot I had removed the optional part earlier. I've fixed it now